src/main/java/sirius/biz/jobs/params/Parameter.java

```diff
@@ -122,6 +148,25 @@ public boolean isVisible(Map<String, String> context) {
         return get(context).isPresent();
     }
 
+    /**
```

Comment:
wouldn't a getter be enough here? it is only used in another condition, we could even use this as a parameter (s.a.)